lib/generators/react_on_rails/templates/no_redux/base/client/app/bundles/HelloWorld/containers/HelloWorldContainer.jsx

```diff
@@ -15,14 +15,12 @@ export default class HelloWorld extends React.Component {
     this.state = { name: this.props.name };
   }
 
-  updateName(name) {
-    this.setState({ name });
-  }
+  updateName = (name) => { this.setState({ name }); };
 
   render() {
     return (
       <div>
-        <HelloWorldWidget name={this.state.name} updateName={e => this.updateName(e)} />
+        <HelloWorld name={this.state.name} updateName={this.updateName} />
```

Comment:
Please note that there will be few changes in no-redux example as HelloWorldWidget.jsx is shared. Also, there's a lint violation here. And I simplified it a little bit with the [arrow function](https://babeljs.io/blog/2015/06/07/react-on-es6-plus#arrow-functions) way.